integration-tests/smoke/reorg_above_finality_test.go

```diff
@@ -15,25 +16,39 @@ import (
 	tc "github.com/smartcontractkit/chainlink/integration-tests/testconfig"
 )
 
-func TestReorgAboveFinality(t *testing.T) {
+func TestReorgAboveFinality_FinalityTagDisabled(t *testing.T) {
 	t.Parallel()
 
 	l := logging.GetTestLogger(t)
-	config, err := tc.GetConfig("Smoke", tc.OCR2)
+	config, err := tc.GetConfig(t.Name(), tc.LogPoller)
 	require.NoError(t, err, "Error getting config")
 
 	privateNetworkConf, err := actions.EthereumNetworkConfigFromConfig(l, &config)
 	require.NoError(t, err)
 
-	nodeFinalityDepthInt := int64(10)
+	// Get values from the node config
+	configMap := make(map[string]interface{})
+	err = toml.Unmarshal([]byte(config.NodeConfig.CommonChainConfigTOML), &configMap)
+	if err != nil {
+		log.Fatalf("Error unmarshaling TOML: %s", err)
+	}
+	nodeFinalityDepthInt, isFinalityDepthSet := configMap["FinalityDepth"].(int64)
+	nodeFinalityTagEnabled := configMap["FinalityTagEnabled"].(bool)
+	l.Info().Int64("nodeFinalityDepth", nodeFinalityDepthInt).Bool("nodeFinalityTagEnabled", nodeFinalityTagEnabled).Msg("Node reorg config")
+
+	var reorgDepth int
+	if isFinalityDepthSet {
+		reorgDepth = int(nodeFinalityDepthInt) + 5
+	} else {
+		reorgDepth = 15
+	}
+	minChainBlockNumberBeforeReorg := reorgDepth + 10
 
 	testEnv, err := test_env.NewCLTestEnvBuilder().
 		WithTestInstance(t).
 		WithTestConfig(&config).
 		WithPrivateEthereumNetwork(privateNetworkConf.EthereumNetworkConfig).
-		WithMockAdapter().
 		WithCLNodes(6).
-		WithFunding(big.NewFloat(*config.Common.ChainlinkNodeFunding)).
```

Comment:
curious, why we don't need node funding?

Reply:
@Tofel We don't do any transactions here